README.md

```diff
@@ -13,6 +13,10 @@ as changes are made to your containers.
 An ECR repository to store the built Docker image will be created for you, if
 one doesn't already exist.
 
+This plugin sets the ECR lifecycle policy to ensure that only 10 images are stored. Additionally, images
+older than 30 days will automatically expire. Automatically expiring images ensures the cache remains somewhat 
+up-to-date with patches.
```

Comment:
A bit of info duplicated between here and L170

Reply:
Good catch, fixed